charts/k8up/test/deployment_test.go

```diff
@@ -37,8 +37,9 @@ func Test_Deployment_ShouldRender_EnvironmentVariables(t *testing.T) {
 	assert.Equalf(t, wantRepo+":"+wantTag, envs[0].Value, "Deployment does not use required Env Value from %s", wantVar)
 	assert.Equal(t, "TZ", envs[1].Name)
 	assert.Equal(t, wantTimezone, envs[1].Value)
-	assert.Equal(t, "BACKUP_OPERATOR_NAMESPACE", envs[3].Name)
-	assert.Equal(t, "metadata.namespace", envs[3].ValueFrom.FieldRef.FieldPath)
+	assert.Equal(t, "BACKUP_SKIP_WITHOUT_ANNOTATION", envs[3].Name)
+	assert.Equal(t, "BACKUP_OPERATOR_NAMESPACE", envs[4].Name)
+	assert.Equal(t, "metadata.namespace", envs[4].ValueFrom.FieldRef.FieldPath)
 	assert.Equal(t, "BACKUP_GLOBALCPU_REQUEST", envs[4].Name, "Deployment does not use configured Env Name")
 	assert.Equal(t, wantCpuRequest, envs[4].Value, "Deployment does not use configured Env Value")
 	assert.Equal(t, "VARIABLE", envs[5].Name, "Deployment does not use configured Env Name")
```

Comment:
You forgot to move these values as well. The whole array shifts by one because of the new variable:
```suggestion
	assert.Equal(t, "BACKUP_GLOBALCPU_REQUEST", envs[5].Name, "Deployment does not use configured Env Name")
	assert.Equal(t, wantCpuRequest, envs[5].Value, "Deployment does not use configured Env Value")
	assert.Equal(t, "VARIABLE", envs[6].Name, "Deployment does not use configured Env Name")
	assert.Equal(t, "VALUE", envs[6].Value, "Deployment does not use configured Env Value")
```

Then the `make chart-test` runs through for me:
```

make chart-test
cd charts && go test ./...
ok      charts/k8up/test        1.554s
ok      charts/k8up/test/prometheus     (cached)
```